webapp/urls.py

```diff
@@ -12,7 +12,8 @@
 
 # Try to find templates
 urlpatterns += [
-    url(r'^search/?$', SearchView.as_view(template_name='search.html')),
+    url(r'^search/?$', RedirectView.as_view(
```

Comment:
I guess maybe you didn't use the [`redirects.yaml`](https://github.com/canonical-websites/docs.ubuntu.com/blob/master/redirects.yaml) because it doesn't redirect with query strings?

If you replace `django_yaml_redirects` used here with [`canonicalwebteam.yaml_responses`](https://github.com/canonical-webteam/yaml-responses#django), then it will include query strings in redirects. This would be a neater solution.